shuup/core/models/_orders.py

```diff
@@ -276,13 +340,39 @@ def ensure_default_statuses(self):
             else:
                 OrderStatus.objects.create(**defaults)
 
+        order_status_qs = OrderStatus.objects.all()
+
+        for order_status in order_status_qs:
+            allowed_status_list = []
+            if order_status.identifier == DefaultOrderStatus.INITIAL.value:
+                allowed_status_list = [
+                    DefaultOrderStatus.PROCESSING.value,
+                    DefaultOrderStatus.COMPLETE.value,
+                    DefaultOrderStatus.CANCELED.value,
+                ]
+            elif order_status.identifier == DefaultOrderStatus.PROCESSING.value:
+                allowed_status_list = [
+                    DefaultOrderStatus.COMPLETE.value,
+                    DefaultOrderStatus.CANCELED.value,
+                ]
+
+            if allowed_status_list:
+                allowed_queryset = OrderStatus.objects.filter(identifier__in=allowed_status_list)
+                order_status.allowed_next_statuses.add(*allowed_queryset)
```

Comment:
So `allowed_next_statuses` is essentially a static list?
Then why make a database entry for it instead of e.g. a `@property` to the `OrderStatus` model that simply returns this same list?